mteb/tasks/Reranking/eng/BIRCO_Reranking.py

```diff
@@ -0,0 +1,211 @@
+# BIRCO_Reranking.py
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+class BIRCORerankingBase(AbsTaskReranking):
+    """
+    Base class for all BIRCO reranking tasks.
+    
+    Expects the Hugging Face dataset to have records with keys:
+      - "query_id": str
+      - "query_text": str
+      - "corpus": list of dicts (each with "corpus_id" and "corpus_text")
+      - "relevance": dict mapping corpus_id to an integer relevance score
+    Also uses an instruction stored in metadata (metadata.instruction).
+    """
+    def dataset_transform(self, hf_dataset):
+        queries = {}
+        corpus = {}
+        qrels = {}
+        instructions = {}
+        
+        # For each record, build the standard dictionaries.
+        for record in hf_dataset:
+            qid = record["query_id"]
+            queries[qid] = record["query_text"]
+            qrels[qid] = record["relevance"]
+            # Use the instruction defined in metadata (if provided); otherwise, use an empty string.
+            instructions[qid] = getattr(self.metadata, "instruction", "")
+            for doc in record["corpus"]:
+                cid = doc["corpus_id"]
+                if cid not in corpus:
+                    corpus[cid] = doc["corpus_text"]
+                    
+        return {"queries": queries, "corpus": corpus, "qrels": qrels, "instructions": instructions}
+
+
+class BIRCODorisMaeReranking(BIRCORerankingBase):
+    metadata = TaskMetadata(
+        name="BIRCO-DorisMae",
+        description=(
+            "Retrieval task using the DORIS-MAE dataset from BIRCO. This dataset contains 60 queries "
+            "that are complex research questions from computer scientists. Each query has a candidate pool of "
+            "approximately 110 abstracts. Relevance is graded from 0 to 2 (scores of 1 and 2 are considered relevant)."
+        ),
+        reference="https://github.com/BIRCO-benchmark/BIRCO",
+        type="Reranking",
+        category="Academic",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg",

```

Comment:
We don't have `ndcg` metric. I think this should be as
```suggestion
        main_score="ndcg_at_10",
```